doc/nrf/ug_radio_coex.rst

```diff
@@ -0,0 +1,196 @@
+attempt to transmit at the same time RF waves will interfere with each other leading to decreased performance and unneeded higher power
+consumption. Also, in some cases (such as receiving ACK) no radio should be transmitting in order to ensure correct reception.
+
+In order to mitigate coexistence issues and improve coexistence performance a Packet Traffic Arbiter (PTA) is used. Both SR and WiFi request access to RF
+and PTA grants/denies that access.
+
+This guide describes how to add WiFi Coexistence (CX) support to your SR application in |NCS|. In order to do it, both
+:ref:`Software support` and :ref:`Hardware description` must be added.
+
+Software support
+****************
+
+If your application uses SR protocols and requires coexistence with WiFi, you must enable :ref:`nrfxlib:mpsl_cx` in the
+:ref:`nrfxlib:mpsl` (MPSL) library.
+
+CX with MPSL
+============
+
+The MPSL library provides an implementation for the WiFi Coexistens module for SR applications.
+To use this implementation, your application must use a protocol driver that enables the WiFi Coexistence (CX) feature.
+The library provides multiprotocol support, but you can also use it in applications that require only one protocol.
+To avoid conflicts, check the protocol documentation to see if the protocol uses CX support provided by MPSL.
+
+.. note::
+   Currently, the following protocols use CX support provided by MPSL:
+
+   * :ref:`ug_thread`
+   * :ref:`ug_zigbee`
+
+.. _ug_radio_cx_requirements:
+
+Enabling MPSL
+-------------
+
+Before you add the devicetree node in your application, complete the following steps:
+
+   The MPSL library provides API to handle signals coming from PTA and controlling requests send to PTA.
+   See :ref:`nrfxlib:mpsl_lib` in the nrfxlib documentation for details.
+
+1. Add support for the MPSL library in your application.
+2. Enable support for MPSL implementation in |NCS| by setting the :kconfig:`CONFIG_MPSL` Kconfig option to ``y``.
+
+Enabling CX
+-------------
+
+In order to enable CX:
+
+1. Set the :kconfig:`CONFIG_MPSL_CX` Kconfig option to ``y``.
+2. Enable support for specific Coex implementation. The currently supported implementations are "generic 3-pin interface" and "thread", for
+   details see :ref:`WiFi Coex implementations`.
+#. On nRF53 devices, you must also apply the same Kconfig options mentioned in step 2 to the network core - see :ref:`_ug_multi_image`.
+
+Hardware description
+********************
+
+The |NCS| provides a wrapper that configures WiFi Coexistence based on devicetree (DTS) and Kconfig information.
+
+To enable CX support for any for the currently supported CX implementation, you must add an ``nrf_radio_coex`` node in the devicetree file, as
+described below. The node can also be provided by the devicetree file of the target development kit or by an overlay file. See
```

Comment:
I hope people will use NCS not only for development kits but also for end products, for their custom boards. So:
`the target development kit` --> `the target board`